controller/QtiCreator.php

```diff
@@ -62,6 +72,11 @@ class QtiCreator extends tao_actions_CommonModule
     use OntologyAwareTrait;
     use HttpJsonResponseTrait;
 
+    private const TEXT_WIDGETS = [
```

Comment:
This constant is not required anymore :)

Reply:
It was done